.github/ISSUE_TEMPLATE/clarification-request.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Clarification request
+about: You’ve found something confusing
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  Please paste (a) link(s) from your browser showing the issue (bare
+  so it can be seen, not buried in a Markdown link).
+
+  Often a clarification request has as much to do with _where_ in the
+  docs something is addressed as _how._ If you think something is
+  useful but in the wrong place(s), or needs to be in multiple places
+  because of where different readers will look, say so!
+
+  If you're not sure where your request for clarification could be
+  best dealt with, either provide the URL where you think it should
+  be, provide multiple possible URLs, or delete the line.
+
+  Please don't provide a URL you wish existed that does not -- that
+  should be in a "New material" request instead.
+
+  -->
+  **Found in URL:** URL-HERE
+<!--
+  Also include the commit hash shown at the bottom of your browser
+  page, so we know what version you’re looking at. Better yet, include
+  the entire “From FILE.pod6 at perl6/doc on GitHub, commit a0b1c2d.”
+  line you’ll find as the first line of every doc page’s footer. -->
+**Found at commit:** COMMIT-OR-FROM-LINE-HERE
+## The problem
+
+<!--
+
+  Please describe the content you feel needs clarification. A quote,
+  cut and pasted, would be much appreciated (use `> ` or a blockquote
+  so we can easily see where it begins and ends).
+
+-->
+
+## Suggestions
+
+
+<!--
+  If applicable, tell us in "Suggestions" what could be done to solve
+  the problem, such as "Rephrase the description" or "Change the
+  example program so it doesn't introduce two unrelated concepts at
+  once."
+
+  Please note we do not have the ability to add diagrams at present
+  aside from the type graphs.
+
+  If you don't know how to fix the issue (this _is_ a documentation
+  site and we expect it to be used by people who are not expert!),
+  please don't worry about exact fixes. However, even a vague
+  description of what would help will help us help you to help
```

Comment:
help help help help?

Reply:
I’m not sure how to respond to a criticism of a joke.... If you think there shouldn’t be humor even in places that aren’t very visible then I’ll rewrite it.

Reply:
The sentence is just hard to read.

Reply:
> The sentence is just hard to read.

If you’re referring to the sentence with the parenthetical—PTAL¹, as I’ve removed it. If you’re referring to the (edited) jokey sentence, _“However, even a vague description of what would help
  will help us help you to help everyone!”_ then I’d just point out that _was_ the joke, and I don’t think it would honestly confuse anyone. But I’ll cut it to, _“...even a vague description of what would help is useful”_ if you truly think it’s confusing in a meaningful way (perhaps to less fluent English speakers?).
—
¹ PTAL: _(acronym)_ Please take another look.